lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -786,17 +787,20 @@ def test_text_search(self, text_search_rewrite):
         })
 
     def test_following(self):
-        self.register_subscribed_threads_response(self.user, [], page=1, num_pages=1)
+        self.register_subscribed_threads_response(self.user, [], page=1, num_pages=0)
         result = get_thread_list(
             self.request,
             self.course.id,
             page=1,
             page_size=11,
             following=True,
-        )
+        ).data
+
+        expected_result = make_paginated_api_response([], 0, 0, None, None)
```

Comment:
These calls are fairly opaque.  When passing so many arguments, better to use kwargs, so we know what they mean, especially when the values passed don't say anything meaningful.  Better to use:

    make_paginated_api_response(results=[], count=0, num_pages=0, next_link=None, previous_link=None)

or even:

    make_paginated_api_response(
        results=[],
        count=0,
        num_pages=0,
        next_link=None,
        previous_link=None,
    )